src/pages/guides/mpc-core-kit.mdx

```diff
@@ -25,235 +25,293 @@ import Tabs from "@theme/Tabs";
 
-const evmProvider = new EthereumSigningProvider({ config: { chainConfig } });
-evmProvider.setupProvider(coreKitInstance);
-```
+### Create an Instance
 
-Here, we're instantiating the Web3Auth MPC CoreKit SDK and using the `chainConfig` property to set
-the chainId and chainNamespace. The `chainId` and `chainNamespace` are the id and the namespace
-respectively of the EVM chain you're connecting to. We've initialized them for the Ethereum Mainnet
-chain for this guide.
+While creating an instance of the SDK, you can define the curve used for account. This is an
+important step since it defines the type of signing that will be used for the transactions, and
+curve for the user's account. The SDK supports `secp256k1` and `ed25519` curves.
 
-Sometimes, you might face network congestion. To avoid this, you can use the `rpcTarget` property to
-specify the URL of the node you want to connect to.
+<CreateAnInstance />
 
-### Initialize the SDK
+### Initialize
 
-```tsx
-await coreKitInstance.init();
-// This will initialize the SDK
+Once you have created an instance of the SDK, you need to initialize it.
 
-// Check the status of the SDK with
-coreKitInstance.status;
+```ts
+// Web3AuthMPCCoreKit instance from previous steps
+await coreKitInstance.init();
 ```
 
 ## Authentication
 
-### Logging In
+As a prerequisite, before authentication, you need to create a verifier for your login method on the
+Web3Auth Dashboard. [Learn how to set up a Custom Verifier](#set-up-a-custom-verifier).
 
-Once initialized, you can use the `loginWithOauth()` or `loginWithJWT()` function to authenticate
-the user when they click the login button. Here, you can use a Single verifier or an Aggregate
-verifier to authenticate the user.
+There are two ways to login your users, depending on the type of authentication method you've
+chosen. If you are looking for an Authentication Flow in your application like
+[Single Page Application(SPA)](https://www.oauth.com/oauth2-servers/single-page-apps/) flow, you can
+use the [loginWithOAuth](#log-in-with-oauth) method.
 
-<Tabs
-defaultValue="single"
-values={[
-  { label: "Single Verifier", value: "single" },
-  { label: "Aggregate Verifier", value: "aggregate" },
-]}
->
-<TabItem value="single">
+If you are looking to pass a JWT-based IdToken to the SDK from your application, like
+[Regular Web Application(RWA)](https://www.oauth.com/oauth2-servers/server-side-apps/) flow or even
+using your own JWT provider, you can use the [loginWithJWT](#log-in-with-jwt) method.
 
-```tsx
-const web3authProvider = await coreKitInstance.loginWithOauth({
-  subVerifierDetails: {
-    typeOfLogin: "google",
-    verifier: "w3a-google-demo", // your verifier name
-    clientId: "519228911939-cri01h55lsjbsia1k7ll6qpalrus75ps.apps.googleusercontent.com", // your client id received from google
-  },
-});
-```
+### Log In With OAuth
+
+To authenticate users using Single Page Application(SPA) flow, you can use the loginWithOAuth
```

Comment:
Let's work on these descriptions to make it beginner friendly, similarly at different places, let's sync for this.